core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.rtl.css

```diff
@@ -0,0 +1,390 @@
+/*
+  NOTE to developers: Only the '.css' files should be edited. The styling should
+  be generalized enough that the autogenerated '.rtl.css' files should work
+  without any manual input.
+  This can be generated by running 'python -m scripts.generate_rtl_css' script.
+  Some additional rules that can be added to the CSS files can be found here:
+  https://rtlcss.com/learn/usage-guide/control-directives .
+*/
+body {
```

Comment:
This seems like a pretty general rule. Why do we have it on this page and not elsewhere?

Reply:
I'm not sure, maybe it is only required for the elements of learner dashboard? This was already present.

Reply:
Hm, just to check -- if you remove it, does the page break? If not then I think you can remove it, and if so then let's leave it in.

Reply:
Removed